src/lib/util/tauri.ts

```
@@ -54,3 +54,13 @@ export async function resetAndClear({
     errorLogger(error);
   }
 }
+
+export async function writeFile(
+  configFullPath: string,
+  config: IConfig
+): Promise<void> {
+  await invoke('create_file', {
+    path: configFullPath,
+    content: JSON.stringify(config, null, 2)
+  }).catch((error) => errorLogger(error));
```

I am changing error handling in another PR, but moving forward instead of catch and log it is better to throw an error and update store, so it is displayed on UI

---

We can update here accordingly if your PR merges first